integration_tests/Cargo.toml

```diff
@@ -33,6 +33,6 @@ tempfile = {workspace = true}
 nft_ingester = { path = "../nft_ingester" }
 postgre-client = { path = "../postgre-client", features = ["integration_tests"] }
 metrics-utils = { path = "../metrics_utils" }
-rocks-db = { path = "../rocks-db" }
+rocks-db = { path = "../rocks-db", features = ["integration_tests"] }
```

Comment:
At a first glance this import looks slightly bizarre to me. Do it mean we import source with the tests attached? If so, maybe we can separate common structures into a dedicated module or something?

Reply:
there is defined method `clean_db` with integration_tests feature, not to build it for usual usage